observability.tf

```diff
@@ -0,0 +1,24 @@
+locals {
+  observability_map = module.dynamic_values.clusters_map
+}
+
+
+##############################################################################
+# Observability Instances (LogDNA + Sysdig)
+##############################################################################
+
+module "observability_instances" {
```

Comment:
I would just leave the observability stuff out for now. It's a large topic on its own, and would require further discussions. Let's just focus this PR on consolidating the base-ocp logic.